addons/point_of_sale/models/pos_config.py

```diff
@@ -479,13 +479,15 @@ def create(self, values):
     def write(self, vals):
         opened_session = self.mapped('session_ids').filtered(lambda s: s.state != 'closed')
         if opened_session:
-            str = []
+            forbidden_fields = []
             for key in self._get_forbidden_change_fields():
                 if key in vals.keys():
-                    str.append(key)
-            if len(str) > 0:
-                raise UserError(
-                    _("Unable to modify this PoS Configuration because you can't modify " + ", ".join(str)) + " while a session is open.")
+                    forbidden_fields.append(key)
```

Comment:
Might be a good idea to get the translated label for the field here if we're going to put the information in a translated message?